### PR TITLE
Added Splash Screen and App icon

### PR DESCRIPTION
**Pull request by ModupeD**
Updated App title and ReadMe with from Nutrition AI to NutriMind.

Updated App Logo and Splash Screen with new App Logo

app splash screen

app icon logo

### CODE REVIEW

File: nutrition_ai/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png
Comment (nathan-wick) [inline nutrition_ai/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png:1]: For these images, could we zoom in and center the logo? That way there's less white space.

**Review by nathan-wick — CHANGES_REQUESTED** [review]
Looks AWESOME! I made one small request, let me know what you think. Other than that, this should be good to merge.

File: nutrition_ai/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png
Comment (ModupeD) [reply nutrition_ai/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png:1]: I like it, thank you. I learned something new

**Review by ModupeD — COMMENTED** [review]
This is good. Thank you

**Review by nathan-wick — APPROVED** [review]
Looks great! Thank you!

**Pull request merged**